docs/contracts/validators-exit-bus-oracle.md

```diff
@@ -0,0 +1,9 @@
+# ValidatorsExitBusOracle
+
+- [Source code](https://github.com/lidofinance/lido-dao/blob/master/contracts/0.8.9/oracle/ValidatorsExitBusOracle.sol)
+- [Deployed contract](https://etherscan.io/address/0x0De4Ea0184c2ad0BacA7183356Aea5B8d5Bf5c6e)
+
+A contract implementing a message bus between the on-chain protocol and off-chain observers
```

Comment:
Between oracle and node operators actually. Both are off-chain.

Reply:
Thank you, agree :+1: